test/topology_custom/test_mv_read_concurrency.py

```diff
@@ -84,3 +84,55 @@ async def do_mv_inserts(i: int):
 
     if failed:
         raise failed
+
+# This test verifies that the writes causing view updates don't make Scylla use excessive memory.
```

Comment:
Note that the wording you use in this comment - and commit messages is confusing. We usually use the phrase "view update" as the process of **updating** the **view**, i.e., writing to the view table. Here you use it mostly in the sense of "doing the work needed to **prepare** a view update", i.e., the read-before-write that happens on the base replica (NOT the view replica!).
When you use a sentence like "the writes causing view updates don't make Scylla use excessive memory." it's very vague because which writes are you talking about (the user's write request? the write to the base table? the writes to the view table?) and there are so many things that take memory.

But I think we've rehashed this PR long enough, so I won't insist on "perfecting" the comments in this PR.